src/Domain/Insights/FixerDecorator.php

```diff
@@ -142,19 +143,14 @@ private function skipFilesFromExcludedFiles(\SplFileInfo $file): bool
             return false;
         }
         foreach ($this->excludedFiles as $excludedFile) {
-            if (self::pathsAreEqual($this->dir . DIRECTORY_SEPARATOR . $excludedFile, $path)) {
+            if (PathComparator::areEqual($this->dir . DIRECTORY_SEPARATOR . $excludedFile, $path)) {
```

Comment:
Let's rename this to just `Paths::areEqual`. It's more readable and allows us to add more than comparison methods in the static class.